### DISCUSSION TITLE
FieldsLinker-master

### DISCUSSION THREAD

**Discussion by jayaprakashmani92** [PhilippeMarcMeyer/FieldsLinker#33] [Q&A]
Right now, there are only two lists connected by a line. If we need to connect multiple lists, what should we do? Thanks

**Comment by PhilippeMarcMeyer** [comment]
Hello, I made another plugin that could meet your needs :
https://github.com/PhilippeMarcMeyer/LinksMaker Cheers

Le mar. 3 sept. 2024, 15:04, jayaprakashmani92 ***@***.***> a
écrit :

**Comment by jayaprakashmani92** [comment]
Hi @PhilippeMarcMeyer 
Thanks for Sharing awesome option. But my requirement in below screenshot. If its possible please let me know procedure. 
Thanks in advance 
![List1-list-4](https://github.com/user-attachments/assets/96f89cd7-b3c1-4631-9586-ea9dc323c0b5)

**Reply by PhilippeMarcMeyer** [reply]
The only option you would have is to author/modify the other plugin I mentioned 

Cheers
